openapi_python_client/resolver/resolved_schema.py

```diff
@@ -0,0 +1,202 @@
+import hashlib
+from typing import Any, Dict, Generator, List, Tuple, Union, cast
+
+from .reference import Reference
+from .resolver_types import SchemaData
+
+
+class ResolvedSchema:
+    def __init__(self, root: SchemaData, refs: Dict[str, SchemaData], errors: List[str]):
+        self._root: SchemaData = root
+        self._refs: Dict[str, SchemaData] = refs
```

Comment:
In last refactor, the keys of refenrences dict changed from: 
`./TSXXXX`, to `/absolute/path/to/TSXXX`, in order to avoid the case where two external ref have a relative reference to them pointing to two different files.

It may need some fix in `ResolvedSchema` class, not tested, not sure if that change introduce behaviour change or not